sources/assets/Xenko.Core.Assets/PackageSessionPublicHelper.cs

```diff
@@ -63,13 +63,11 @@ private static void CheckMSBuildToolset()
             // Check that we can create a project
             using (var projectCollection = new Microsoft.Build.Evaluation.ProjectCollection())
             {
-                if (projectCollection.Toolsets.Any(x => new Version(x.ToolsVersion).Major >= 15))
+                if (!projectCollection.Toolsets.Any(x => new Version(x.ToolsVersion).Major >= 15))
```

Comment:
Cool!
Thinking about it, since it's a string, I wouldn't want it to break the second they add a semantic name or use something that's not a proper version (`new Version()` throws an exception if invalid input).

What about using `PackageVersion.TryParse(x.ToolsVersion, out var version) && version.Version.Major >= 15` as condition instead?